MIGRATION_v2-to-v3.md

```diff
@@ -25,14 +25,45 @@ or `import Channel from "@sendbird/uikit-react/Channel"`
 
 ### Changes
   * Removed renderCustomMessage, renderChatItem. Use renderMessage instead
-  ```example
+  Current channel can be obtained from `useChannel()`
+  Message related actions should be implemented using sendbirdSelectors
+  ```jsx
+    <Channel renderMessage={MyAdminMessageComponent} />
```

Comment:
Why don't we take a user message as an example?
Because admin messages are not enough to use in introducing our features.

Reply:
This is typo, sorry